lib/appoptics_apm/config.rb

```diff
@@ -64,6 +64,30 @@ def self.load_config_file
         config_files.each { |path| $stderr.puts "  #{path}" }
       end
       load(config_files[0])
+      check_env_vars
+    end
+
+    # There are 4 variables that can be set in the config file or as env vars.
+    # since env vars take priority we need to check them here.
+    # Oboe reads the env vars, so we need to set them, if they are defined via config file
+    def self.check_env_vars
+      ENV['APPOPTICS_SERVICE_KEY'] ||= AppOpticsAPM::Config[:service_key]
```

Comment:
For the service key in particular, it is helpful for a user to know if the value from the config was used or if the environment variable was used.